### PR TITLE
Back to a connection.

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: R/importWQP.R [hunk 1]
```diff
@@ -57,8 +57,9 @@ importWQP <- function(obs_url, zip=TRUE, tz=""){
   if(!file.exists(obs_url)){
     
     if(zip){
+      message("zip encoding access still in development")
       temp <- tempfile()
-      # options(timeout = 120)
+      options(timeout = 120)
```
Comment (jread-usgs) [inline R/importWQP.R]: would be nice to allow the user to specify this for known bigger requests?

Comment (ldecicco-USGS) [inline R/importWQP.R]: If I remember correctly, 120 is as big as you can go. I'll check

Comment (ldecicco-USGS) [inline R/importWQP.R]: 120 was specifically chosen back in the day for a reason...it's not a random number.

Comment (ldecicco-USGS) [inline R/importWQP.R]: Well, now I can't find the reason for 120. For the record, I don't think it's quitting if the download hasn't finished in 120 seconds (plenty of mine take longer than that). I think it's if the service hasn't somehow pinged at least 1 bit back in 120 seconds.

Comment (jread-usgs) [inline R/importWQP.R]: gotcha

Comment (ldecicco-USGS) [inline R/importWQP.R]: You have led me down a rabbit hole where I'm trying to figure out the difference between R option's timeout and curlSetOpt(timeout = 200)

```
curlSetOpt(timeout = 120)
#vs:
options(timeout = 120)
```
@jiwalker-usgs any insight?

Comment (jiwalker-usgs) [inline R/importWQP.R]: I have no idea what the difference is here, but I think timeout refers to the readTimeout (how long after no data has been received until stopping).  Both probably do the same thing, but options may apply to other packages.

**Pull request merged**